doc/api/process.md

```diff
@@ -129,7 +129,8 @@ unforeseen and unpredictable issues.
 
 Exceptions thrown from within the event handler will not be caught. Instead the
 process will exit with a non zero exit code and the stack trace will be printed.
-This is to avoid infinite recursion.
+This is to avoid infinite recursion. There may be other situations where the
```

Comment:
Some clarification on the "other situations" would be nice. After reading this, people might not be clear whether or not their `uncaughtException` handler can be expected to run or not.